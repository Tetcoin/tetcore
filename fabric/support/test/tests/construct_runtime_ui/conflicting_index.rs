use fabric_support::construct_runtime;

construct_runtime! {
	pub enum Runtime where
		UncheckedExtrinsic = UncheckedExtrinsic,
		Block = Block,
		NodeBlock = Block,
	{
		System: system::{},
		Noble1: noble1::{} = 0,
	}
}

fn main() {}
