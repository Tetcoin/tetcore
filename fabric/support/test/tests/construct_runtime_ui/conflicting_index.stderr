error: Module indices are conflicting: Both modules System and Noble1 are at index 0
 --> $DIR/conflicting_index.rs:9:3
  |
9 |         System: system::{},
  |         ^^^^^^

error: Module indices are conflicting: Both modules System and Noble1 are at index 0
  --> $DIR/conflicting_index.rs:10:3
   |
10 |         Noble1: noble1::{} = 0,
   |         ^^^^^^^
